test/bench/bench.cpp

```diff
@@ -110,9 +108,9 @@ struct benchmark_case
 };
 
 
-void load_benchmark(fs::path path)
+void load_benchmark(fs::path path, const fs::path& subdir)
```

Comment:
why pass one `fs::path` by value and another by const ref?

Reply:
Because the one passed by value is modified inside the function. I will change it though because it also negatively affects the readability in the place where it is modified.